Files/ViewModels/SettingsViewModel.cs

```diff
@@ -730,6 +730,18 @@ public SortOption DefaultDirectorySortOption
             set => Set((byte)value);
         }
 
+        public bool PreviewPaneEnabled
+        {
+            get => Get(false);
+            set => Set(value);
+        }
+
+        public bool EnableAdaptivePreviewPane
+        {
+            get => Get(true);
+            set => Set(value);
+        }
```

Comment:
Can you move these to the preferences region and add xml to improve IntelliSense?